rotkehlchen/exchanges/kraken.py

```diff
@@ -153,7 +153,14 @@ def history_event_from_kraken(
                 else:
                     event_subtype = HistoryEventSubType.RECEIVE
             elif event_type == HistoryEventType.STAKING:
-                event_subtype = HistoryEventSubType.REWARD
+                # in the case of ETH.S after the merge rewards don't compound unlike what happens
+                # with other assets and a virtual event with negative amount is created
+                if asset == A_ETH2 and raw_amount < ZERO:
```

Comment:
what about events before withdrawals activation? Would we have any negative events then?

Reply:
Sorry I wrote merge but is the withdrawals activation what triggered it. Will fix the comment. Before the withdrawals the events are correct